packages/manager/core/request-tagger/src/requestTagger.ts

```diff
@@ -13,6 +13,16 @@ export enum Header {
   VERSION = 'X-OVH-MANAGER-VERSION',
 }
 
+export enum Route {
+  Listing = 'listing',
+  Dashboard = 'dashboard',
+  Onboarding = 'onboarding',
+  New = 'New',
+}
+
+export const getRequestTaggerPciPath = (project: string, route?: Route) =>
```

Comment:
For a pci project app at the root, we have to create a new function or update the params cause the page name is like this 'pci.projects'
I prefer to add the complete page name in 'id' of routes.tsx directly without the declaration of enum, cause for each new route, dev don't have to update this enum.
This packages is used by more than 40 apps , if we update this file every time we have to redeploy it in each

Reply:
You can put this logic in a pci-specific package